packages/js-deployer/bin/deployer

```diff
@@ -469,8 +469,44 @@ const pluginsToUpdate = [
     to: `$1${options.feeDynamicBytesMultiPayment}`
   },
   {
-    from: /(dynamicFees: {[a-zA-Z0-9: ,\n{}]*addonBytes: {[a-zA-Z0-9: ,\n{}]*delegateResignation: )([0-9]+)/g,
+    from: /(dynamicFees: {[a-zA-Z0-9: ,\n{}]*addonBytes: {[a-zA-Z0-9: ,\n{}]*delegateResignation: )([0-9]+)(,)/g,
     to: `$1${options.feeDynamicBytesDelegateResignation}`
+  },
+  {
+    from: /(htlcLock: )([0-9]+)(,\n)/g,
```

Comment:
can you make these match like the ones above it? so including `dynamicFees` and `addonBytes`